share/completions/pygmentize.fish

```diff
@@ -0,0 +1,30 @@
+function __fish_print_pygmentize --argument-names type description
+    for line in (pygmentize -L $type | sed -e "s%^\* \(.*\):%\1%;tx;d;:x" | string split ", ")
```

Comment:
```
> pygmentize -L filters | string match -r "\* (.*):"
* raiseonerror:
raiseonerror
* gobble:
gobble
* whitespace:
whitespace
* highlight:
highlight
* tokenmerge:
tokenmerge
* codetagify:
codetagify
* keywordcase:
keywordcase
```

It returns what I want and the entire line. I don't know how to get rid of it (without doing something horrible). :-/